code/src/sixsq/nuvla/server/resources/spec/nuvlabox_status_0.cljc

```diff
@@ -241,8 +241,26 @@
              :json-schema/order 24)))
 
 
+(s/def ::power-consumption-metric
```

Comment:
Any reason to not have a map instead of the array? 
We already do that for disks attribute for example

Reply:
don't remember why I did it this way. Changed already